models/DataObject/ClassDefinition/Data.php

```diff
@@ -1199,6 +1199,104 @@ public function getUnique()
         return false;
     }
 
+    /**
+     * @param mixed $data
+     * @param DataObject\Concrete|DataObject\Localizedfield|DataObject\Objectbrick\Data\AbstractData|DataObject\Fieldcollection\Data\AbstractData $object
+     * @param array $params
+     *
+     * @throws \Exception
+     */
+    protected function setDataToObject($data, $object, $params = [])
```

Comment:
jost one thought: maybe mark it as internal and move it to AbstractRelations as it is only used there.

Reply:
marked it internal